src/main/java/at/hannibal2/skyhanni/features/event/hoppity/HoppityEggsManager.kt

```diff
@@ -27,9 +27,14 @@ object HoppityEggsManager {
 
     val config get() = SkyHanniMod.feature.event.hoppityEggs
 
+    /**
+     * REGEX-TEST: §d§lHOPPITY'S HUNT §r§dYou found a §r§9Chocolate Lunch Egg §r§don a ledge next to the stairs up§r§d!
+     * REGEX-TEST: §d§lHOPPITY'S HUNT §r§dYou found a §r§aChocolate Dinner Egg §r§dbehind Emissary Sisko§r§d!
+     * REGEX-TEST: §d§lHOPPITY'S HUNT §r§dYou found a §r§9Chocolate Lunch Egg §r§dnear the Diamond Essence Shop§r§d!
+     */
     private val eggFoundPattern by ChocolateFactoryAPI.patternGroup.pattern(
         "egg.found",
-        "§d§lHOPPITY'S HUNT §r§dYou found a §r§.Chocolate (?<meal>\\w+) Egg.*"
+        "§d§lHOPPITY'S HUNT §r§dYou found a §r§.Chocolate (?<meal>\\w+) Egg §r§d(?<note>.*)§r§d!"
```

Comment:
will not match this:
§d§lHOPPITY'S HUNT §r§dYou found a §r§9Chocolate Lunch Egg §r§dnear the §r§5Dark Auction§r§d!
or
§d§lHOPPITY'S HUNT §r§dYou found a §r§6Chocolate Breakfast Egg §r§din §r§cUnincorporated Territory§r§d!

Reply:
> i dont understand

these two messages dont match your new pattern

Reply:
wait never mind i'm going crazy